lib/ocpp1_6/charge_point_state_machine.cpp

```diff
@@ -1,6 +1,14 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2020 - 2022 Pionix GmbH and Contributors to EVerest
 #include <ocpp1_6/charge_point_state_machine.hpp>
+#include <ocpp1_6/enums.hpp>
+
+#include <stddef.h>
+#include <cstdint>
+#include <stdexcept>
+#include <utility>
+
+#include <fsm/async.hpp>
```

Comment:
why does this for example gets included here again, if it is already included in the charge_point_state_machine.hpp?

Reply:
Good question, I tried to apply some suggestions from include-what-you-use but might have gone a little overboard with some of them :D